src/native/monodroid/xamarin_getifaddrs.cc

```diff
@@ -876,7 +879,7 @@ get_link_address (const struct nlmsghdr *message, struct _monodroid_ifaddrs **if
 				}
 
 				if (payload_size > 0) {
-					size_t alloc_size = ADD_WITH_OVERFLOW_CHECK (size_t, payload_size, room_for_trailing_null);
```

Comment:
the move away from the `ADD_WITH_OVERFLOW_CHECK()` macro to `Helpers::add_with_overflow_check<T>()` is also an unmentioned change.  Why is it part of this PR?

Reply:
It's not mentioned because it doesn't change anything in the way this code **works**, it's functionally equivalent to the macro.  The reason for his change is twofold - first, I much prefer functions than preprocessor macros, since they're typesafe.  Second, this way I can take advantage of the `source_location` header and report the code position in a much cleaner, type-safe and better way.  We had the macro before only because this was the only way to report on the `source:line` location (taking advantage of the `__FILE__` and `__LINE__` preprocessor macros, respectively)